### DISCUSSION TITLE
2022-08: Iteration 7 汇报

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#337] [Announcements]
# 2022-08: Iteration 7 汇报

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-08/

**Comment by PsiACE** [comment]
差点看成千分之一 :) 总之，Xuanwo 哥哥大成功

**Reply by Xuanwo** [reply]
:sob: :sob: :sob:

**Reply by sundy-li** [reply]
赞，“羞愧离职” 搞得太严重了吧，哈哈。

这个主要原因是databend 还没有持续到性能做监控，后面补足这块后，要合并到主干的性能回退就可以里面侦测出来了。

**Comment by howbbig** [comment]
哈哈，发现错别字，“高诉迭代”

**Reply by Xuanwo** [reply]
感谢，已经修复了
